docs/install-manuals/publishing-keycloak.md

```diff
@@ -0,0 +1,13 @@
+# Making keycloak production ready
+
+By default, Misarch creates a few test accounts inside the `Misarch` realm with easy logins.\
+In particular, those are
+
+- `admin`
+- `seller`
+- `test`
+
+You should delete these accounts before publishing the platform to keep the permission system intact.
+
+Furthermore, you should change the password of the **Keycloak** `admin` account in the `master` realm so that no external person can access the Keycloak console.\
```

Comment:
Is there a way to require a password reset, to make it secure by default?

Reply:
Yes, but I've explicitly disabled it